.envrc

```diff
@@ -1,3 +1,7 @@
 use asdf
 
-dotenv
\ No newline at end of file
+dotenv
+
+# used in dockercompose and dockerfile 
+export BUILDPLATFORM=$(uname -m)
```

Comment:
this is passed to docker compose, which passes it to the dockerfile, which uses it to install tableau deps or not.